packages/web/src/scss/components/Collapse/index.html

```diff
@@ -152,21 +150,65 @@ <h2 class="docs-Heading">Collapse with hide trigger when expanded </h2>
     >
       ...more
     </a>
-    <div
+    <span
       id="CollapseExampleIII"
       class="Collapse"
     >
-      <div class="Collapse__content">
+      <span class="Collapse__content">
         Commodo metus a lorem, a aliquet vestibulum rutrum pharetra sapien sed, ullamcorper quis odio dolor ut aliquam. Rutrum suspendisse, fermentum tellus metus a lorem
         cursus volutpat proin bibendum, sed diam a duis id dui et tempus. Ligula non, sapien augue libero eget aliquam semper varius, posuere urna leo vitae ullamcorper.
         Venenatis posuere, sapien rhoncus dolor quis semper porttitor et vestibulum, odio leo ac nibh molestie placerat. Tellus suspendisse, dui lectus tellus quis eget
         sollicitudin dolor sit amet, dui ac nibh nulla mauris. Feugiat sollicitudin, integer bibendum at sem sapien non nullam sed id tortor, suscipit fusce gravida nec eget.
-      </div>
-    </div>
+      </span>
+    </span>
   </p>
 
 </section>
 
+<section class="docs-Section">
+
+  <h2 class="docs-Heading">Default with helper class for changing content inside toggle</h2>
+
+  <p>
+    Ante rhoncus, tristique nunc et elit id in ex sem consectetur id, consectetur et elit nibh sed ac. Sed ac convallis, sem lobortis convallis et nisi sapien in ex sem ullamcorper
+    iaculis, aliquet sem lobortis pellentesque sit amet. Congue nulla congue, porttitor quis orci quam elit rutrum scelerisque nibh maximus, felis a integer maximus praesent.
+    Metus varius, at sem dui vivamus elementum luctus nisl sit amet dolor sit amet, mauris et iaculis nec fermentum. Augue sapien, nisl vel purus nullam odio morbi a fusce vitae
+    mattis vitae, posuere tristique bibendum aliquam. In donec dolor ut, imperdiet quam fermentum molestie vulputate scelerisque ac nec, tortor mi orci sollicitudin elementum.
+  </p>
+  <div
+    id="CollapseExampleV"
+    class="Collapse"
+  >
+    <div class="Collapse__content">
+      Aliquam varius, consequat posuere a lacinia mauris eu tellus condimentum ut id ante, accumsan vehicula nulla neque. Mauris mi orci, in donec nullam odio leo sapien et vehicula nunc a
+      lacinia, fermentum arcu ullamcorper posuere. Mauris euismod, ac nec ante fermentum praesent nisi commodo neque placerat, vivamus dui et tempus pulvinar suspendisse. Porttitor eget,
+      sollicitudin hendrerit bibendum nulla aliquam sit amet leo vitae, eget consectetur diam a vestibulum. Adipiscing lorem ipsum, arcu condimentum posuere semper morbi condimentum dui,
+      bibendum nunc aenean facilisis. Phasellus euismod, donec sem odio ligula praesent finibus nibh convallis, tristique aliquam sed id tortor sem lobortis.
+    </div>
+  </div>
+  <div class="mb-400">
+    <button
+      type="button"
+      data-toggle="collapse"
+      data-target="CollapseExampleV"
+      class="Button Button--primary Button--medium"
+    >
+      <span class="accessibility-open">Show less</span>
+      <span class="accessibility-closed">Show more</span>
+    </button>
+    &nbsp;
```

Comment:
A tiny detail: As we already discussed elsewhere, it's no shame to use our utility classes for offsets in our previews. You would do the same in a real-world project 🙂.